File: backend/zvms/impls/student.py

import typing


StudentVolbookByStuIdResponse = TypedDict('StudentVolbookByStuIdResponse',
            {
                "rec": typing.Sequence[structs.VolunteerRecord]
            }
        )
        
def getStudentVolbookByStuId() -> StudentVolbookByStuIdResponse : '''
## GET student/volbook/<stuId>
# 查询某个学生的义工本

''' 

File: web/src/apis/fApi/student.ts
import { get, post } from "./utils";
import * as structs from "../types/structs";

/**
 * ## GET student/volbook/<stuId>
 * 查询某个学生的义工本

 */
function getStudentVolbookByStuId(): Promise<{
  rec: Array<structs.VolunteerRecord>;
}> {
  return get("student/volbook/<stuId>", {});
}
